pkg/tap/ip_pool.go

```diff
@@ -50,7 +50,7 @@ func (p *IPPool) GetOrAssign(mac string) (net.IP, error) {
 
 	var i uint64
 	for i = 1; i < p.count; i++ {
-		candidate, err := cidr.Host(p.base, int(i))
+		candidate, err := cidr.Host(p.base, int(i)) //nolint:gosec
```

Comment:
In this case, we want to know if there’s an overflow, but maybe it’s better to report this in `NewIPPool` (but this is up to you)